include/libavcodec/avcodec.pxd

```diff
@@ -116,6 +116,92 @@ cdef extern from "libavcodec/avcodec.h" nogil:
         AVDISCARD_NONKEY
         AVDISCARD_ALL
 
+    cdef enum AVChromaLocation:
```

Comment:
Why all the new enums?

Reply:
Sorry, just saw this. My GitHub notifications are a mess. 

I had to add them because of the declared types in this method: https://github.com/PyAV-Org/PyAV/blob/1213f02dc28a14aa803753a8b57f9eae9e07604a/include/libavcodec/avcodec.pxd#L533